public/blogPosts/fall2018/hackschool-session-3-frontend/index.md

```diff
@@ -727,48 +727,48 @@ Your `updateClock` function should look like this:
 
 ```javascript
 function updateClock() {
-  let date = new Date();
-  let hours = date.getHours();
-  let minutes = date.getMinutes();
-  let seconds = date.getSeconds();
-  let ampm = '';
-
-  let color = 'rgb(' + time2color(hours, minutes, seconds) + ')';
-
-  if (hours >= 12) {
-    ampm = 'PM';
-  } else {
-    ampm = 'AM';
-  }
-
-  if (seconds < 10) {
-    seconds = '0' + seconds;
-  }
-  if (minutes < 10) {
-    minutes = '0' + minutes;
-  }
-
-  if (hours != 12) {
-    hours = hours % 12;
-  }
-
-  if (hours == 0) {
-    hours = 12;
-  }
-
-  time.innerHTML = hours + ':' + minutes + ':' + seconds + ' ' + ampm;
-
-  hex.innerHTML = color;
-
-  if (colorIsBg) {
-    time.style.color = 'white';
-    hex.style.color = 'white';
-    body.style.backgroundColor = color;
-  } else {
-    body.style.backgroundColor = 'white';
-    time.style.color = color;
-    hex.style.color = color;
-  }
+	let date = new Date();
```

Comment:
nit: will this extra spacing show up on the blog? is it still readable? if so, then feel free to ignore this comment. Otherwise, you might need to revert these tabs back to spaces.